hw/ip_templates/clkmgr/data/clkmgr.hjson.tpl

```diff
@@ -477,7 +477,7 @@ rg_srcs = get_rg_srcs(typed_clocks)
         If the hardware block is idle, then the clock is disabled.
         If the hardware block is not idle, the clock is kept on.
 
-        For the enable case, the software hint is immediately honored and the clock turned on.  Hardware does not provide any
+        For the enable case, the software hint is immediately honoured and the clock turned on.  Hardware does not provide any
```

Comment:
Ooh. I think this one might not be needed. (Americans have no honour but have "honor" instead...)